README.md

```diff
@@ -123,7 +123,7 @@ Install dependencies:
 
 Clone source code, note you'll need the `default` branch:
 
-    hg clone http://bitbucket.org/ignitionrobotics/ign-gui -b default
+    git clone http://github.com/ignitionrobotics/ign-gui -b master
```

Comment:
update branch name 

This commit in ign-rendering is relevant:
https://github.com/ignitionrobotics/ign-rendering/pull/62/commits/de66fe4d728221496f814f700d6b8fe7c31b1c1f